docs/HowToGuides/GettingStarted.md

```diff
@@ -366,14 +373,14 @@ several more steps are necessary to set up this environment:
 * For a Ninja target that you want to build (e.g. `swift-frontend`), add a
   target to the empty project, using the _External Build System_ template.
 * In the _Info_ pane of the target settings, set
-  * _Build Tool_ to the path of the `ninja` executable (the output of
+  * _Build Tool_ to the absolute path of the `ninja` executable (the output of
     `which ninja` on the command line)
   * _Arguments_ to the Ninja target name (e.g. `swift-frontend`)
-  * _Directory_ to the path of the build directory associated with the Ninja
-    target. For Swift targets, including the standard library and runtime, you
-    want `path/to/swift-project/build/Ninja-*/swift-macosx-*`
-* Add a scheme for the target. Be careful not to select a target from one of the
-  generated Xcode projects.
+  * _Directory_ to the absolute path of the directory where the Ninja target
+    lives. For Swift targets (the compiler, standard library, runtime, and
+    related tooling), this is the `build/Ninja-*/swift-macosx-*` directory.
+* Add a scheme for the target. In the drop-down menu, be careful not to mistake
+  your target for a similar target from one of the generated Xcode projects.
```

Comment:
```suggestion
  your target for a similar target from one of the generated Xcode project's.
```

Reply:
By "from" I really mean "belonging to". I guess the latter is the better option after all.

Reply:
I think I misunderstood the sentence previously. The part that felt wrong to me was basically the plural "projects", but now in just realised the build script does create more than 1 project.